UX-Design/UX Newsletters.md

```diff
@@ -0,0 +1,10 @@
+##Newsletters
+
+1. [Mailchimp UX Newsletter](http://www.theuxnewsletter.com/)
```

Comment:
Please also include details/brief about each links, for example https://github.com/PareshMayani/Learning-Resources/blob/master/android/Useful_links.md